includes/definitions/allied.yaml

```diff
@@ -1,10 +1,16 @@
 os: allied
-text: AlliedWare
+text: 'Alliedware'
 type: network
+icon: allied
+group: allied
 ifname: 1
 over:
     - { graph: device_bits, text: Traffic }
+#exclude ...207.1.4.126 for ATI 8000S/8000GS which identify as "radlan" OS. 
```

Comment:
You can put this next to line it's intended for with a # to show it's a comment or just remove it.

Reply:
K thanks.